pyclashbot/__main__.py

```diff
@@ -3,12 +3,21 @@
 from __future__ import annotations
 
 import os
+import warnings
 from os.path import expandvars, join
 from typing import TYPE_CHECKING, Any
 
+warnings.filterwarnings( # mutes localization error from console (no functionality loss)
```

Comment:
can we do this in `ui.py` to localize where we import `ttkbootstrap`?

i think having this co-located where it matters is better than shoving it in main